src/format.rs

```diff
@@ -22,40 +59,70 @@ pub fn run_lengths(
     let mut i = 0;
     let mut match_start: bool = false;
     while i < aln.len() {
-	match_start = (aln[i] != '-' && aln[i] != ' ') && !match_start;
-	if match_start {
-	    let start = i;
-	    let mut matches: usize = 0;
-	    while i < aln.len() && (aln[i] != '-' && aln[i] != ' ') {
-		matches += (aln[i] == 'M' || aln[i] == 'R') as usize;
-		i += 1;
-	    }
-	    encodings.push((start + 1, i, matches, i - start - matches));
-	    match_start = false;
-	} else {
-	    i += 1;
-	}
+        match_start = (aln[i] != '-' && aln[i] != ' ') && !match_start;
+        if match_start {
+            let start = i;
+            let mut matches: usize = 0;
+            while i < aln.len() && (aln[i] != '-' && aln[i] != ' ') {
+                matches += (aln[i] == 'M' || aln[i] == 'R') as usize;
+                i += 1;
+            }
+            encodings.push((start + 1, i, matches, i - start - matches));
+            match_start = false;
+        } else {
+            i += 1;
+        }
     }
     encodings
 }
 
+/// Format a refined translation relative to the reference sequence.
+///
+/// Jointly reads nucleotides from the reference sequence `ref_seq` and the
+/// character representation of an alignment `alignment` to determine the
+/// nucleotide sequence of the alignment relative to the reference.
+///
+/// Only works with refined translations as the input (TODO verify).
```

Comment:
A new Todo was discovered. If it is not a priority right now,consider marking it for later attention.
TODO verify).
- [ ] Ignore